disable-breaking-updates.py

```diff
@@ -23,18 +23,18 @@
 try:
     with settings_path.open() as settings_file:
         settings = json.load(settings_file)
+except IOError:
+    settings_path.parent.mkdir(parents = True, exist_ok = True)
```

Comment:
Just some style nits:

```suggestion
    settings_path.parent.mkdir(parents=True, exist_ok=True)
```